Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,237 @@
+            }
+
+            // Find similar example for find similar faces from faceIds with similar confidence.
+            IList<SimilarFace> similarResults = Client.Face.FindSimilarAsync(faceIds[0].Value, null, null, faceIds).Result;
+            if (similarResults.Count == 0)
+            {
+                Console.WriteLine($"Not found the same with `{faceIds[0].Value}` from similar result.");
+            }
+            foreach (var similarResult in similarResults)
+            {
+                Console.WriteLine(
+                    similarResult.Confidence>0.5
+                        ? $"Faces from {faceIds[0].Value} & {similarResult.FaceId} are of the same (Positive) person, similarity confidence: {similarResult.Confidence}."
+                        : $"Faces from {faceIds[0].Value} & {similarResult.FaceId} are of different (Negative) persons, similarity confidence: {similarResult.Confidence}.");
+            }
+
+            Console.WriteLine("\nPress ENTER to exit.");
+        }
+
+        public static async Task FindSimilar_in_FaceList()
+        {
+            List<string> targetImageFileNames =
+                            new List<string> { "Family1-Dad1.jpg", "Family1-Daughter1.jpg", "Family1-Mom1.jpg", "Family1-Son1.jpg", "Family2-Lady1.jpg", "Family2-Man1.jpg", "Family3-Lady1.jpg", "Family3-Man1.jpg" };
+            string sourceImageFileName = "findsimilar.jpg";
+
+            // Create a face list.
+            string faceListId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create FaceList {faceListId} {Environment.NewLine}");
+            await Client.FaceList.CreateAsync(faceListId, faceListId, "face list for sample");
+
+            foreach (var targetImageFileName in targetImageFileNames)
+            {
+                // Read image file.
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Add face to face list.
+                    Console.WriteLine($"Add faces to face list from image `{targetImageFileName}`.");
+                    var faces = await Client.FaceList.AddFaceFromStreamAsync(faceListId, stream, targetImageFileName);
+                    if (faces == null)
+                    {
+                        Console.WriteLine($"[Error] No persisted face from image `{targetImageFileName}`.");
+                        return;
+                    }
+                }
+            }
+
+            // Get persisted faces.
+            List<PersistedFace> persistedFaces = Client.FaceList.GetAsync(faceListId).Result.PersistedFaces as List<PersistedFace>;
+
+            // Read image file.
+            using (FileStream stream = new FileStream(Path.Combine("Images", sourceImageFileName), FileMode.Open))
+            {
+                // Detect faces from image stream.
+                IList<DetectedFace> detectedFaces = Client.Face.DetectWithStreamAsync(stream).Result;
```

Comment:
Please replace all `.Result` by `await` in this PR and later PRs inside `async Task` functions.

Reply:
done